scripts/release_info.py

```diff
@@ -37,6 +41,7 @@
 # pylint: enable=wrong-import-position
 
 GIT_CMD_GET_STATUS = 'git status'
+GIT_CMD_CHECKOUT = 'git checkout -- %s'
```

Comment:
For any string that still needs interpolation, start its name with GIT_CMD_TEMPLATE_

Reply:
Done.